pipeline/lookup_domain.py

```diff
@@ -0,0 +1,2029 @@
+    "www.victoriassecret.com": 1915,
+    "www.voanews.com": 1916,
+    "www.volcanomail.com": 1917,
+    "www.walmart.com": 1918,
+    "www.warchild.org": 1919,
+    "www.warhammeronline.com": 1920,
+    "www.washingtontimes.com": 1921,
+    "www.webbox.com": 1922,
+    "www.wechat.com": 1923,
+    "www.weebly.com": 1924,
+    "www.weforum.org": 1925,
+    "www.wfp.org": 1926,
+    "www.wftucentral.org": 1927,
+    "www.whatsapp.com": 1928,
+    "www.whitepower.com": 1929,
+    "www.who.int": 1930,
+    "www.whs.mil": 1931,
+    "www.wiesenthal.com": 1932,
+    "www.wikia.com": 1933,
+    "www.wikidata.org": 1934,
+    "www.wikipedia.org": 1935,
+    "www.wiktionary.org": 1936,
+    "www.winespectator.com": 1937,
+    "www.wix.com": 1938,
+    "www.wluml.org": 1939,
+    "www.wmtransfer.com": 1940,
+    "www.wnd.com": 1941,
+    "www.womenonwaves.org": 1942,
+    "www.womenonweb.org": 1943,
+    "www.womensmediacenter.com": 1944,
+    "www.womenwarpeace.org": 1945,
+    "www.wordreference.com": 1946,
+    "www.worldbank.org": 1947,
+    "www.worldcoalition.org": 1948,
+    "www.worldhealth.net": 1949,
+    "www.worldjewishcongress.org": 1950,
+    "www.worldlingo.com": 1951,
+    "www.worldometers.info": 1952,
+    "www.worldpulse.com": 1953,
+    "www.worldrtd.net": 1954,
+    "www.worldwildlife.org": 1955,
+    "www.wzo.org.il": 1956,
+    "www.xbox.com": 1957,
+    "www.xenu.net": 1958,
+    "www.xinhuanet.com": 1959,
+    "www.xinhua.org": 1960,
+    "www.xroxy.com": 1961,
+    "www.xvideos.com": 1962,
+    "www.xxlmag.com": 1963,
+    "www.yahoo.com": 1964,
+    "www.yelp.com": 1965,
+    "www.ymca.int": 1966,
+    "www.yola.com": 1967,
+    "www.youporn.com": 1968,
+    "www.youtube.com": 1969,
+    "www.zanzu.nl": 1970,
+    "www.zeit.de": 1971,
+    "www.zensur.freerk.com": 1972,
+    "www.zoho.com": 1973,
+    "www.zone-h.org": 1974,
+    "xabbs.com": 1975,
+    "xanga.com": 1976,
+    "xbox.com": 1977,
+    "xda-developers.com": 1978,
+    "xe.com": 1979,
+    "xfinity.com": 1980,
+    "xhamster.com": 1981,
+    "xiami.com": 1982,
+    "xiaohongshu.com": 1983,
+    "xinhuanet.com": 1984,
+    "xnxx.com": 1985,
+    "xvideos.com": 1986,
+    "xxx.lanl.gov": 1987,
+    "y2mate.com": 1988,
+    "yahoo.co.jp": 1989,
+    "yahoo.com": 1990,
+    "yandex.com": 1991,
+    "yandex.ru": 1992,
+    "yelp.com": 1993,
+    "yoox.com": 1994,
+    "youku.com": 1995,
+    "youm7.com": 1996,
+    "youporn.com": 1997,
+    "youtube.com": 1998,
+    "yt1s.com": 1999,
+    "ytmp3.cc": 2000,
+    "yts.mx": 2001,
+    "yumpu.com": 2002,
+    "yy.com": 2003,
+    "zalo.me": 2004,
+    "zdf.de": 2005,
+    "zerodha.com": 2006,
+    "zhibo8.cc": 2007,
+    "zh.wikipedia.org": 2008,
+    "zillow.com": 2009,
+    "zippyshare.com": 2010,
+    "ziprecruiter.com": 2011,
+    "zoho.com": 2012,
+    "zol.com.cn": 2013,
+    "zoomshare.com": 2014,
+    "zoom.us": 2015
+}
+
+DOMAIN_PARTITIONS = 100
+
+
+def domain_partition(domain: str) -> int:
```

Comment:
Yes, the buckets contain 11 domains - 33 domains. I also tried increasing the number of partitions to 150 and 200, in case the size of the largest bucket was causing the problem.